ChangeLog

```diff
@@ -3,7 +3,7 @@ Mbed TLS ChangeLog (Sorted per branch, date)
 = Mbed TLS 3.2.1 branch released 2022-07-12
 
 Bugfix
-   *  Re-add missing generated file library/ssl_debug_helpers_generated.c
+   *  Re-add missing generated file library/psa_crypto_driver_wrappers.c
```

Comment:
I find it a bit weird to modify a changelog entry for a past release. Especially since the bug is specifically in the release and doesn't apply to the released product. Yes, this sentence was wrong, and the patch makes it correct. But what's the point of fixing the 3.2.1 changelog entry in the 3.3.0 release? Should the 3.3.0 release even have a changelog entry for 3.2.1? (Typically not, because x.y.1 wouldn't be an ancestor of x.(y+1).0, but in this particular case the need for a patch release was discovered very quickly and it is an ancestor.)

Reply:
I would say that it might be beneficial for people that try to find any past references to `psa_crypto_driver_wrappers.c` not by looking at commit history but by grepping the repo. Some time ago I was looking for information about a feature added to pylint, and while I was downloading the code I also opened the changelog.